examples/nodejs/cache/basic.ts

```diff
@@ -1,36 +1,133 @@
-import {CacheGet, CreateCache, CacheSet, CacheClient, Configurations, CredentialProvider} from '@gomomento/sdk';
```

Comment:
i just included this file so you can see how i was testing things and playing with the API. these changes won't go into the final PR.